AUTHORS

```diff
@@ -526,3 +526,4 @@ Carsten Brandt <mail@cebe.cc>
 Marcin Szczepanski <marcin@imagichine.com.au>
 Josh Clow <josh@textio.com>
 Jakub Holy <jakubholy@jakubholy.net>
+Marcos Scriven <marcos@scriven.org>
```

Comment:
Please don't add yourself to AUTHORS (that happens automatically during releases)

Reply:
Removed.